### ISSUE TITLE
Chart fails to load all sets after loading 0 values

### ISSUE THREAD

**Issue by alpriest** [patrykandpatrick/vico#488]
### How to reproduce

See code below.

Press the "load 2 sets" button and see two sets of values are plotted correctly.
Press the "load 1 set of 0 value" button and see that no sets are plotted.
Press the "load 2 sets" button again and see that only 1 set of vales are plotted.

@Composable
fun Greeting(viewModel: MyViewModel = MyViewModel()) {
    Chart(
        chart = columnChart(),
        chartModelProducer = viewModel.producer,
        modifier = Modifier
            .width(500.dp)
            .height(500.dp)
    )

    Column {
        Button(onClick = { viewModel.load1() }) {
            Text("Load 2 sets")
        }

        Button(onClick = { viewModel.load2() }) {
            Text("Load 1 set of 0 value")
        }
    }
}

class MyViewModel : ViewModel() {
    val producer: ChartEntryModelProducer = ChartEntryModelProducer()

    fun load1() {
        producer.setEntries(
            listOf(
                listOf(
                    FloatEntry(5f, 10f),
                    FloatEntry(10f, 20f),
                    FloatEntry(15f, 15f),
                    FloatEntry(20f, 18f),
                    FloatEntry(25f, 8f),
                    FloatEntry(30f, 28f),
                ),
                listOf(
                    FloatEntry(5f, 10f),
                    FloatEntry(10f, 20f),
                    FloatEntry(15f, 15f),
                    FloatEntry(20f, 18f),
                    FloatEntry(25f, 8f),
                    FloatEntry(30f, 28f),
                )
            )
        )
    }

    fun load2() {
        producer.setEntries(
            listOf(
                listOf(
                    FloatEntry(5f, 0f),
                    FloatEntry(10f, 0f),
                    FloatEntry(15f, 0f),
                    FloatEntry(20f, 0f),
                    FloatEntry(25f, 0f),
                    FloatEntry(30f, 0f),
                )
            )
        )
    }
}

@Preview(showBackground = true)
@Composable
fun GreetingPreview() {
    GraphTestTheme {
        Greeting()
    }
}

### Observed behavior

After plotting a set of values of y 0, the multiple plot sets never recover and only 1 set are shown.

### Expected behavior

Everytime we request multiple sets of data then they are show regardless of what happened previously.

### Vico version(s)

Latest stable version

### Android version(s)

minsdk 29, targetsdk 34

### Additional information

Viso dependency is com.patrykandpatrick.vico:compose-m3:1.13.1
Bug is repoducible in the Android Studio compose preview panel.

**Comment by patrickmichalik** [comment]
Hello! Thanks for the report. The problem seems to be that, for a data set whose every _y_ value is zero, `ChartValues.minY` and `ChartValues.maxY` are both zero by default. Their difference is zero, which leads to division by zero, and so things fall apart. We’ll update the logic such that the default _y_ range is [0, 1] for such data. In the meantime, to fix the problem, you can use an `AxisValuesOverrider` implementation along these lines:

```
object : AxisValuesOverrider<ChartEntryModel> {
    override fun getMaxY(model: ChartEntryModel) = if (model.maxY != 0) model.maxY else 1f
}
```

For `MergeMode.Stack`, replace `model.maxY` with `model.stackedPositiveY`.

**Comment by alpriest** [comment]
Thanks, just dropped that in and it worked a treat 👍

**Comment by patrickmichalik** [comment]
I’m glad to hear that. You’re welcome!

**Comment by patrickmichalik** [comment]
[Vico 2.0.0 Alpha 4](https://github.com/patrykandpatrick/vico/releases/tag/v2.0.0-alpha.4) introduces the improvement mentioned above.

**Comment by patrickmichalik** [comment]
This will be resolved in Vico 1 too—[Vico 1.14.0 Alpha 1](https://github.com/patrykandpatrick/vico/releases/tag/v1.14.0-alpha.1) includes the fix.

**Comment by patrickmichalik** [comment]
[Vico 1.14.0](https://github.com/patrykandpatrick/vico/releases/tag/v1.14.0) is rolling out. Cheers!
